documentation/modules/non-admin-permissions-for-ui.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: PROCEDURE
+
+[id="non-admin-permissions_{context}"]
+= Granting permissions to non-administrators to work with migration plan components
+
+As an administrator, you can work with all components of migration plans (for example, providers, network mappings, and migration plans).
+
+By default, non-administrators have limited ability to work with migration plans and their components. You can modify their roles to allow them full access to all components or you can give them limited permissions.
+
+For example, you can grant non-administrators the following set of permissions:
+
+* Create and modify storage maps, network maps, and migration plans for the namespaces they have access to.
+* Attach providers created by adminstrators to storage maps, network maps, and migration plans.
+
+[cols="3", options="header"]
+.Permissions required for non-adminstrators to work with migration plan components but not create providers
+|===
+|Actions |API groups |Resources
```

Comment:
the `All` part is too broad since it gives "admin" permissions, we can change it with:
view and edit roles for plans (plans.forklift.konveyor.io-v1beta1-view and plans.forklift.konveyor.io-v1beta1-edit), ... view permissions on providers (providers.forklift.konveyor.io-v1beta1-view) ...